tests/ert/unit_tests/gui/simulation/test_run_dialog.py

```diff
@@ -708,3 +708,46 @@ def test_that_stdout_and_stderr_buttons_react_to_file_content(
 
         with qtbot.waitSignal(run_dialog.accepted, timeout=30000):
             run_dialog.close()
+
+
+@pytest.mark.integration_test
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.parametrize(
+    "design_matrix_entry",
+    (True, False),
+)
+def test_that_design_matrix_show_parameters_button_is_enabled(
```

Comment:
Here we are also testing that the button should be disabled, but I cannot think of a better name.
Maybe we can add to the test that when you push the button, the design matrix param viewer should pop up?

Reply:
Hm, like in another test or inside this one?

Reply:
We should really keep the unit tests at a unit level, but here I would test it in the same one. Then we would get to test the button visibility and the click functionality. We could call the test `test_design_matrix_show_parameters_button`

Reply:
I'd need to create a pandas frame and it'd become integration I think. But maybe it's still fine to have as unit though :shrug:

Reply:
Ah, you are right. Let's keep it as is.

Reply:
> Ah, you are right. Let's keep it as is.

I'll give it a go :smile_cat: